desktop/assets/article.coffee

```diff
@@ -4,7 +4,7 @@ sd = require('sharify').data
 layout = sd.ARTICLE?.layout
 
 $ ->
-  if location.pathname.match('article')
+  if location.pathname.match('/article/|/video/|/series/')
```

Comment:
Quick note that the client needs to be rehydrated to get client-side interactions working like analytics/clicks and this is where we make that available.